projects/packages/publicize/src/class-publicize.php

```diff
@@ -239,7 +253,18 @@ public function get_all_connections_for_user() {
 					$user_id = (int) $connection['connection_data']['user_id'];
 					// phpcs:ignore WordPress.PHP.YodaConditions.NotYoda
 					if ( $user_id === 0 || $this->user_id() === $user_id ) {
-						$connections_to_return[ $service_name ][ $id ] = $connection;
+						if ( is_callable( array( self::class, 'use_amin_ui_v1' ) ) && self::use_admin_ui_v1() ) {
```

Comment:
As it is in the same file and added in the same change, do we need to check if it's callable?

Reply:
Oh good point. My mind was perhaps thinking that the code was written in a plugin 😄

Reply:
Fixed ✅